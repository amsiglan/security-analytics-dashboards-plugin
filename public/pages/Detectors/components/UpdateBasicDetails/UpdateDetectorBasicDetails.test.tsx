/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 */

import React from 'react';
import { render } from '@testing-library/react';
import props from '../../../../models/Detectors/components/UpdateDetectorBasicDetails/UpdateDetectorBasicDetails.mock';
import { expect } from '@jest/globals';
import { UpdateDetectorBasicDetails } from './UpdateBasicDetails';

describe('<UpdateDetectorBasicDetails /> spec', () => {
  it('renders the component', () => {
    const view = render(<UpdateDetectorBasicDetails {...props} />);
    expect(view).toMatchSnapshot();
  });
});
