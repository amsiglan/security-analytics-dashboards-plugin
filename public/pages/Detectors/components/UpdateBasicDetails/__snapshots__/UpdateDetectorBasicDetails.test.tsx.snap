// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UpdateDetectorBasicDetails /> spec renders the component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <h3
          class="euiTitle euiTitle--medium"
        >
          Edit detector details
        </h3>
        <div
          class="euiSpacer euiSpacer--xxl"
        />
        <div
          class="euiPanel euiPanel--paddingMedium euiPanel--borderRadiusMedium euiPanel--plain euiPanel--hasShadow"
          style="padding-left: 0px; padding-right: 0px;"
        >
          <div
            class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsCenter euiFlexGroup--justifyContentSpaceBetween euiFlexGroup--directionRow euiFlexGroup--responsive"
            style="padding: 0px 10px;"
          >
            <div
              class="euiFlexItem"
            >
              <h3
                class="euiTitle euiTitle--medium"
              >
                Detector details
              </h3>
            </div>
          </div>
          <hr
            class="euiHorizontalRule euiHorizontalRule--full euiHorizontalRule--marginXSmall"
          />
          <div
            style="padding: 0px 10px;"
          >
            <div
              class="euiSpacer euiSpacer--m"
            />
            <div
              class="euiFormRow"
              id="some_html_id-row"
            >
              <div
                class="euiFormRow__labelWrapper"
              >
                <label
                  aria-invalid="false"
                  class="euiFormLabel euiFormRow__label"
                  for="some_html_id"
                >
                  <div
                    class="euiText euiText--small"
                  >
                    <strong>
                      Name
                    </strong>
                  </div>
                </label>
              </div>
              <div
                class="euiFormRow__fieldWrapper"
              >
                <div
                  class="euiFormControlLayout"
                >
                  <div
                    class="euiFormControlLayout__childrenWrapper"
                  >
                    <input
                      class="euiFieldText"
                      data-test-subj="define-detector-detector-name"
                      id="some_html_id"
                      placeholder="Enter a name for the detector."
                      required=""
                      type="text"
                      value=""
                    />
                  </div>
                </div>
              </div>
            </div>
            <div
              class="euiSpacer euiSpacer--m"
            />
            <div
              class="euiFormRow"
              id="some_html_id-row"
            >
              <div
                class="euiFormRow__labelWrapper"
              >
                <label
                  aria-invalid="false"
                  class="euiFormLabel euiFormRow__label"
                  for="some_html_id"
                >
                  <div
                    class="euiText euiText--small"
                  >
                    <strong>
                      Description
                    </strong>
                    <i>
                       - optional 
                    </i>
                  </div>
                </label>
              </div>
              <div
                class="euiFormRow__fieldWrapper"
              >
                <textarea
                  class="euiTextArea euiTextArea--resizeVertical euiTextArea--compressed"
                  data-test-subj="define-detector-detector-description"
                  id="some_html_id"
                  placeholder="Enter a description for the detector."
                  rows="3"
                />
              </div>
            </div>
          </div>
        </div>
        <div
          class="euiSpacer euiSpacer--xl"
        />
        <div
          class="euiPanel euiPanel--paddingMedium euiPanel--borderRadiusMedium euiPanel--plain euiPanel--hasShadow"
          style="padding-left: 0px; padding-right: 0px;"
        >
          <div
            class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsCenter euiFlexGroup--justifyContentSpaceBetween euiFlexGroup--directionRow euiFlexGroup--responsive"
            style="padding: 0px 10px;"
          >
            <div
              class="euiFlexItem"
            >
              <h3
                class="euiTitle euiTitle--medium"
              >
                Data source
              </h3>
            </div>
          </div>
          <hr
            class="euiHorizontalRule euiHorizontalRule--full euiHorizontalRule--marginXSmall"
          />
          <div
            style="padding: 0px 10px;"
          >
            <div
              class="euiSpacer euiSpacer--m"
            />
            <div
              class="euiFormRow"
              id="some_html_id-row"
            >
              <div
                class="euiFormRow__labelWrapper"
              >
                <label
                  aria-invalid="true"
                  class="euiFormLabel euiFormRow__label euiFormLabel-isInvalid"
                  for="some_html_id"
                >
                  <div
                    class="euiText euiText--small"
                  >
                    <strong>
                      Select or input source indexes or index patterns
                    </strong>
                  </div>
                </label>
              </div>
              <div
                class="euiFormRow__fieldWrapper"
              >
                <div
                  aria-describedby="some_html_id-error-0"
                  aria-expanded="false"
                  aria-haspopup="listbox"
                  class="euiComboBox"
                  data-test-subj="define-detector-select-data-source"
                  role="combobox"
                >
                  <div
                    class="euiFormControlLayout"
                  >
                    <div
                      class="euiFormControlLayout__childrenWrapper"
                    >
                      <div
                        class="euiComboBox__inputWrap euiComboBox__inputWrap--noWrap euiComboBox__inputWrap-isLoading euiComboBox__inputWrap-isClearable"
                        data-test-subj="comboBoxInput"
                        tabindex="-1"
                      >
                        <p
                          class="euiComboBoxPlaceholder"
                        >
                          Select an input source for the detector.
                        </p>
                        <div
                          class="euiComboBox__input"
                          style="font-size: 14px; display: inline-block;"
                        >
                          <input
                            aria-controls=""
                            data-test-subj="comboBoxSearchInput"
                            id="some_html_id"
                            role="textbox"
                            style="box-sizing: content-box; width: 2px;"
                            value=""
                          />
                          <div
                            style="position: absolute; top: 0px; left: 0px; visibility: hidden; height: 0px; overflow: scroll; white-space: pre; font-family: -webkit-small-control; letter-spacing: normal; text-transform: none;"
                          />
                        </div>
                      </div>
                      <div
                        class="euiFormControlLayoutIcons euiFormControlLayoutIcons--right"
                      >
                        <span
                          class="euiLoadingSpinner euiLoadingSpinner--medium"
                        />
                        <button
                          aria-label="Open list of options"
                          class="euiFormControlLayoutCustomIcon euiFormControlLayoutCustomIcon--clickable"
                          data-test-subj="comboBoxToggleListButton"
                          type="button"
                        >
                          EuiIconMock
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  aria-live="polite"
                  class="euiFormErrorText euiFormRow__text"
                  id="some_html_id-error-0"
                >
                  Select an input source.
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="euiSpacer euiSpacer--xl"
        />
        <div
          class="euiPanel euiPanel--paddingMedium euiPanel--borderRadiusMedium euiPanel--plain euiPanel--hasShadow"
          style="padding-left: 0px; padding-right: 0px;"
        >
          <div
            class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsCenter euiFlexGroup--justifyContentSpaceBetween euiFlexGroup--directionRow euiFlexGroup--responsive"
            style="padding: 0px 10px;"
          >
            <div
              class="euiFlexItem"
            >
              <h3
                class="euiTitle euiTitle--medium"
              >
                Detector schedule
              </h3>
            </div>
          </div>
          <hr
            class="euiHorizontalRule euiHorizontalRule--full euiHorizontalRule--marginXSmall"
          />
          <div
            style="padding: 0px 10px;"
          >
            <div
              class="euiFormRow"
              id="some_html_id-row"
            >
              <div
                class="euiFormRow__labelWrapper"
              >
                <label
                  class="euiFormLabel euiFormRow__label"
                  for="some_html_id"
                >
                  <div
                    class="euiText euiText--small"
                  >
                    <strong>
                      Frequency
                    </strong>
                  </div>
                </label>
              </div>
              <div
                class="euiFormRow__fieldWrapper"
              >
                <div
                  class="euiFormControlLayout"
                >
                  <div
                    class="euiFormControlLayout__childrenWrapper"
                  >
                    <select
                      class="euiSelect"
                      id="some_html_id"
                    >
                      <option
                        value="interval"
                      >
                        By interval
                      </option>
                    </select>
                    <div
                      class="euiFormControlLayoutIcons euiFormControlLayoutIcons--right"
                    >
                      <span
                        class="euiFormControlLayoutCustomIcon"
                      >
                        EuiIconMock
                      </span>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="euiFormRow"
              id="some_html_id-row"
            >
              <div
                class="euiFormRow__labelWrapper"
              >
                <label
                  class="euiFormLabel euiFormRow__label"
                  for="some_html_id"
                >
                  <div
                    class="euiText euiText--small"
                  >
                    <strong>
                      Run every
                    </strong>
                  </div>
                </label>
              </div>
              <div
                class="euiFormRow__fieldWrapper"
              >
                <div
                  class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--directionRow euiFlexGroup--responsive"
                  id="some_html_id"
                >
                  <div
                    class="euiFlexItem"
                  >
                    <div
                      class="euiFormControlLayout"
                    >
                      <div
                        class="euiFormControlLayout__childrenWrapper"
                      >
                        <input
                          class="euiFieldNumber euiFieldNumber--withIcon"
                          data-test-subj="detector-schedule-number-select"
                          min="1"
                          type="number"
                          value="1"
                        />
                        <div
                          class="euiFormControlLayoutIcons"
                        >
                          <span
                            class="euiFormControlLayoutCustomIcon"
                          >
                            EuiIconMock
                          </span>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    class="euiFlexItem"
                  >
                    <div
                      class="euiFormControlLayout"
                    >
                      <div
                        class="euiFormControlLayout__childrenWrapper"
                      >
                        <select
                          class="euiSelect"
                          data-test-subj="detector-schedule-unit-select"
                        >
                          <option
                            value="MINUTES"
                          >
                            Minutes
                          </option>
                          <option
                            value="HOURS"
                          >
                            Hours
                          </option>
                          <option
                            value="DAYS"
                          >
                            Days
                          </option>
                        </select>
                        <div
                          class="euiFormControlLayoutIcons euiFormControlLayoutIcons--right"
                        >
                          <span
                            class="euiFormControlLayoutCustomIcon"
                          >
                            EuiIconMock
                          </span>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="euiSpacer euiSpacer--xl"
        />
        <div
          class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--justifyContentFlexEnd euiFlexGroup--directionRow euiFlexGroup--responsive"
        >
          <div
            class="euiFlexItem euiFlexItem--flexGrowZero"
          >
            <button
              class="euiButton euiButton--primary euiButton-isDisabled"
              disabled=""
              type="button"
            >
              <span
                class="euiButtonContent euiButton__content"
              >
                <span
                  class="euiButton__text"
                >
                  Cancel
                </span>
              </span>
            </button>
          </div>
          <div
            class="euiFlexItem euiFlexItem--flexGrowZero"
          >
            <button
              class="euiButton euiButton--primary euiButton-isDisabled"
              data-test-subj="save-basic-details-edits"
              disabled=""
              type="button"
            >
              <span
                class="euiButtonContent euiButton__content"
              >
                <span
                  class="euiButton__text"
                >
                  Save changes
                </span>
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <h3
        class="euiTitle euiTitle--medium"
      >
        Edit detector details
      </h3>
      <div
        class="euiSpacer euiSpacer--xxl"
      />
      <div
        class="euiPanel euiPanel--paddingMedium euiPanel--borderRadiusMedium euiPanel--plain euiPanel--hasShadow"
        style="padding-left: 0px; padding-right: 0px;"
      >
        <div
          class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsCenter euiFlexGroup--justifyContentSpaceBetween euiFlexGroup--directionRow euiFlexGroup--responsive"
          style="padding: 0px 10px;"
        >
          <div
            class="euiFlexItem"
          >
            <h3
              class="euiTitle euiTitle--medium"
            >
              Detector details
            </h3>
          </div>
        </div>
        <hr
          class="euiHorizontalRule euiHorizontalRule--full euiHorizontalRule--marginXSmall"
        />
        <div
          style="padding: 0px 10px;"
        >
          <div
            class="euiSpacer euiSpacer--m"
          />
          <div
            class="euiFormRow"
            id="some_html_id-row"
          >
            <div
              class="euiFormRow__labelWrapper"
            >
              <label
                aria-invalid="false"
                class="euiFormLabel euiFormRow__label"
                for="some_html_id"
              >
                <div
                  class="euiText euiText--small"
                >
                  <strong>
                    Name
                  </strong>
                </div>
              </label>
            </div>
            <div
              class="euiFormRow__fieldWrapper"
            >
              <div
                class="euiFormControlLayout"
              >
                <div
                  class="euiFormControlLayout__childrenWrapper"
                >
                  <input
                    class="euiFieldText"
                    data-test-subj="define-detector-detector-name"
                    id="some_html_id"
                    placeholder="Enter a name for the detector."
                    required=""
                    type="text"
                    value=""
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            class="euiSpacer euiSpacer--m"
          />
          <div
            class="euiFormRow"
            id="some_html_id-row"
          >
            <div
              class="euiFormRow__labelWrapper"
            >
              <label
                aria-invalid="false"
                class="euiFormLabel euiFormRow__label"
                for="some_html_id"
              >
                <div
                  class="euiText euiText--small"
                >
                  <strong>
                    Description
                  </strong>
                  <i>
                     - optional 
                  </i>
                </div>
              </label>
            </div>
            <div
              class="euiFormRow__fieldWrapper"
            >
              <textarea
                class="euiTextArea euiTextArea--resizeVertical euiTextArea--compressed"
                data-test-subj="define-detector-detector-description"
                id="some_html_id"
                placeholder="Enter a description for the detector."
                rows="3"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        class="euiSpacer euiSpacer--xl"
      />
      <div
        class="euiPanel euiPanel--paddingMedium euiPanel--borderRadiusMedium euiPanel--plain euiPanel--hasShadow"
        style="padding-left: 0px; padding-right: 0px;"
      >
        <div
          class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsCenter euiFlexGroup--justifyContentSpaceBetween euiFlexGroup--directionRow euiFlexGroup--responsive"
          style="padding: 0px 10px;"
        >
          <div
            class="euiFlexItem"
          >
            <h3
              class="euiTitle euiTitle--medium"
            >
              Data source
            </h3>
          </div>
        </div>
        <hr
          class="euiHorizontalRule euiHorizontalRule--full euiHorizontalRule--marginXSmall"
        />
        <div
          style="padding: 0px 10px;"
        >
          <div
            class="euiSpacer euiSpacer--m"
          />
          <div
            class="euiFormRow"
            id="some_html_id-row"
          >
            <div
              class="euiFormRow__labelWrapper"
            >
              <label
                aria-invalid="true"
                class="euiFormLabel euiFormRow__label euiFormLabel-isInvalid"
                for="some_html_id"
              >
                <div
                  class="euiText euiText--small"
                >
                  <strong>
                    Select or input source indexes or index patterns
                  </strong>
                </div>
              </label>
            </div>
            <div
              class="euiFormRow__fieldWrapper"
            >
              <div
                aria-describedby="some_html_id-error-0"
                aria-expanded="false"
                aria-haspopup="listbox"
                class="euiComboBox"
                data-test-subj="define-detector-select-data-source"
                role="combobox"
              >
                <div
                  class="euiFormControlLayout"
                >
                  <div
                    class="euiFormControlLayout__childrenWrapper"
                  >
                    <div
                      class="euiComboBox__inputWrap euiComboBox__inputWrap--noWrap euiComboBox__inputWrap-isLoading euiComboBox__inputWrap-isClearable"
                      data-test-subj="comboBoxInput"
                      tabindex="-1"
                    >
                      <p
                        class="euiComboBoxPlaceholder"
                      >
                        Select an input source for the detector.
                      </p>
                      <div
                        class="euiComboBox__input"
                        style="font-size: 14px; display: inline-block;"
                      >
                        <input
                          aria-controls=""
                          data-test-subj="comboBoxSearchInput"
                          id="some_html_id"
                          role="textbox"
                          style="box-sizing: content-box; width: 2px;"
                          value=""
                        />
                        <div
                          style="position: absolute; top: 0px; left: 0px; visibility: hidden; height: 0px; overflow: scroll; white-space: pre; font-family: -webkit-small-control; letter-spacing: normal; text-transform: none;"
                        />
                      </div>
                    </div>
                    <div
                      class="euiFormControlLayoutIcons euiFormControlLayoutIcons--right"
                    >
                      <span
                        class="euiLoadingSpinner euiLoadingSpinner--medium"
                      />
                      <button
                        aria-label="Open list of options"
                        class="euiFormControlLayoutCustomIcon euiFormControlLayoutCustomIcon--clickable"
                        data-test-subj="comboBoxToggleListButton"
                        type="button"
                      >
                        EuiIconMock
                      </button>
                    </div>
                  </div>
                </div>
              </div>
              <div
                aria-live="polite"
                class="euiFormErrorText euiFormRow__text"
                id="some_html_id-error-0"
              >
                Select an input source.
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="euiSpacer euiSpacer--xl"
      />
      <div
        class="euiPanel euiPanel--paddingMedium euiPanel--borderRadiusMedium euiPanel--plain euiPanel--hasShadow"
        style="padding-left: 0px; padding-right: 0px;"
      >
        <div
          class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsCenter euiFlexGroup--justifyContentSpaceBetween euiFlexGroup--directionRow euiFlexGroup--responsive"
          style="padding: 0px 10px;"
        >
          <div
            class="euiFlexItem"
          >
            <h3
              class="euiTitle euiTitle--medium"
            >
              Detector schedule
            </h3>
          </div>
        </div>
        <hr
          class="euiHorizontalRule euiHorizontalRule--full euiHorizontalRule--marginXSmall"
        />
        <div
          style="padding: 0px 10px;"
        >
          <div
            class="euiFormRow"
            id="some_html_id-row"
          >
            <div
              class="euiFormRow__labelWrapper"
            >
              <label
                class="euiFormLabel euiFormRow__label"
                for="some_html_id"
              >
                <div
                  class="euiText euiText--small"
                >
                  <strong>
                    Frequency
                  </strong>
                </div>
              </label>
            </div>
            <div
              class="euiFormRow__fieldWrapper"
            >
              <div
                class="euiFormControlLayout"
              >
                <div
                  class="euiFormControlLayout__childrenWrapper"
                >
                  <select
                    class="euiSelect"
                    id="some_html_id"
                  >
                    <option
                      value="interval"
                    >
                      By interval
                    </option>
                  </select>
                  <div
                    class="euiFormControlLayoutIcons euiFormControlLayoutIcons--right"
                  >
                    <span
                      class="euiFormControlLayoutCustomIcon"
                    >
                      EuiIconMock
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="euiFormRow"
            id="some_html_id-row"
          >
            <div
              class="euiFormRow__labelWrapper"
            >
              <label
                class="euiFormLabel euiFormRow__label"
                for="some_html_id"
              >
                <div
                  class="euiText euiText--small"
                >
                  <strong>
                    Run every
                  </strong>
                </div>
              </label>
            </div>
            <div
              class="euiFormRow__fieldWrapper"
            >
              <div
                class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--directionRow euiFlexGroup--responsive"
                id="some_html_id"
              >
                <div
                  class="euiFlexItem"
                >
                  <div
                    class="euiFormControlLayout"
                  >
                    <div
                      class="euiFormControlLayout__childrenWrapper"
                    >
                      <input
                        class="euiFieldNumber euiFieldNumber--withIcon"
                        data-test-subj="detector-schedule-number-select"
                        min="1"
                        type="number"
                        value="1"
                      />
                      <div
                        class="euiFormControlLayoutIcons"
                      >
                        <span
                          class="euiFormControlLayoutCustomIcon"
                        >
                          EuiIconMock
                        </span>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="euiFlexItem"
                >
                  <div
                    class="euiFormControlLayout"
                  >
                    <div
                      class="euiFormControlLayout__childrenWrapper"
                    >
                      <select
                        class="euiSelect"
                        data-test-subj="detector-schedule-unit-select"
                      >
                        <option
                          value="MINUTES"
                        >
                          Minutes
                        </option>
                        <option
                          value="HOURS"
                        >
                          Hours
                        </option>
                        <option
                          value="DAYS"
                        >
                          Days
                        </option>
                      </select>
                      <div
                        class="euiFormControlLayoutIcons euiFormControlLayoutIcons--right"
                      >
                        <span
                          class="euiFormControlLayoutCustomIcon"
                        >
                          EuiIconMock
                        </span>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="euiSpacer euiSpacer--xl"
      />
      <div
        class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--justifyContentFlexEnd euiFlexGroup--directionRow euiFlexGroup--responsive"
      >
        <div
          class="euiFlexItem euiFlexItem--flexGrowZero"
        >
          <button
            class="euiButton euiButton--primary euiButton-isDisabled"
            disabled=""
            type="button"
          >
            <span
              class="euiButtonContent euiButton__content"
            >
              <span
                class="euiButton__text"
              >
                Cancel
              </span>
            </span>
          </button>
        </div>
        <div
          class="euiFlexItem euiFlexItem--flexGrowZero"
        >
          <button
            class="euiButton euiButton--primary euiButton-isDisabled"
            data-test-subj="save-basic-details-edits"
            disabled=""
            type="button"
          >
            <span
              class="euiButtonContent euiButton__content"
            >
              <span
                class="euiButton__text"
              >
                Save changes
              </span>
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
